src/app/components/BuildBanner/index.tsx

```diff
@@ -19,9 +19,5 @@ export const BuildBanner: FC = () => {
     ? t('banner.buildStaging')
     : t('banner.buildPreview', { appTitle: getAppTitle() })
 
-  return (
-    <Alert variant="warning-filled" sticky>
-      {message}
-    </Alert>
-  )
+  return <Alert variant="warning-filled">{message}</Alert>
```

Comment:
sticky prop applies different styles (not related to positioning only)

Reply:
:scream: